Cargo.toml

```diff
@@ -17,8 +18,12 @@ travis-ci = { repository = "liranringel/structure" }
 appveyor = { repository = "liranringel/structure" }
 
 [dependencies]
-proc-macro-hack = "0.4"
 structure-macro-impl = { version = "0.1.2", path = "structure-macro-impl" }
-byteorder = "1"
+byteorder = { version = "1", default-features = false }
+core2 = "0.3"
```

Comment:
`core2` has a default `std` feature too, needs `default-features = false`.

Reply:
Fixed in 49eedeb.